### PR TITLE
List Divider (Multiline): add examples and styles

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/1058
[Problem] Missing examples and implementation
[Solution] Add examples and implementation

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
Divider position without multiline has wrong position. Created issue for that https://github.com/Samsung/TAU/issues/1141

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 1]
```diff
@@ -238,27 +238,51 @@
...
+					&.li-has-multiline {
+						min-height: 108 * @unit_base;
 
+						* {
+							text-align: center;
+						}
+
+						& > div:first-child {
+							display: grid;
+							width: 222 * @unit_base;
```
Comment (lslachciak) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: can we stick to px_base?

**Review by lslachciak — CHANGES_REQUESTED** [review]
In your example toggle switch is broken
Please compare:
image
with
image
button on the toggle switch is outside of switch area

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 1 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: Done.

**Comment by grzegorz-czajkowski** [comment]
@lslachciak fixed toggle switch for multi line layout by removing unnecessary top: 2px

**Review by lslachciak — APPROVED** [review]

**Pull request merged**